snapcraft.yaml

```diff
@@ -31,6 +31,7 @@ description: |-
 
  Supported configuration options for the snap (snap set lxd [<key>=<value>...]):
 
+   - apparmor.unprivileged_restrictions_disable: Whether to disable restrictions on unprivileged user namespaces [default=true]
```

Comment:
I believe you are using hyphens now?

Reply:
@tomponline I tested from the up to date repo but had not push. So test results are still good and I've since force pushed.